### DISCUSSION TITLE
Baseline definition for text styles

### DISCUSSION THREAD

**Discussion by bmooij-lg** [ingrammicro/puzzle-tokens#50] [Q&A]
In sketch you can influence the baseline of a text field (via Menu > Text > Baseline)
It provides options such as Default, Superscript, subscript, but also simply raise and lower.

But this can't be applied directly to text styles via the sketch UI. Could this be done using puzzle-tokens?
As I'm having an issue with a font dropping almost to the bottom of the textbox when I define the font-size and line-height via puzzle-tokens. (It's font specific, related to how sketch handles typography)

**Comment by MaxBazarov** [comment]
Sorry for the delay. Checking...

**Comment by MaxBazarov** [comment]
Currently it's not supported by puzzle-tokens. But seems to developer discussions it can be implemented. 
Just need a time to do it.

**Reply by bmooij-lg** [reply]
Please let me know if you can find the time :)

**Comment by MaxBazarov** [comment]
Yep. Resumed the research today.

**Reply by bmooij-lg** [reply]
Let me know if there is anything I can do to help, as this would be a massive improvement to our setup if it can be done.

**Comment by MaxBazarov** [comment]
I will try to restore the problem context tomorrow and inform you.
